source/material_model/perplex_simpler.cc

```diff
@@ -0,0 +1,225 @@
+                             Patterns::Double (0),
+                             "The value of the minimum temperature used to query PerpleX. "
+                             "Units: $K$.");
+          prm.declare_entry ("Maximum material temperature", "6000.",
+                             Patterns::Double (0),
+                             "The value of the maximum temperature used to query PerpleX. "
+                             "Units: $K$.");
+          prm.declare_entry ("Minimum material pressure", "1.e5",
+                             Patterns::Double (0),
+                             "The value of the minimum pressure used to query PerpleX. "
+                             "Units: $Pa$.");
+          prm.declare_entry ("Maximum material pressure", "1.e12",
+                             Patterns::Double (0),
+                             "The value of the maximum pressure used to query PerpleX. "
+                             "Units: $Pa$.");
+
+        }
+        prm.leave_subsection();
+      }
+      prm.leave_subsection();
+    }
+
+
+
+    template <int dim>
+    void
+    PerpleXSimpler<dim>::parse_parameters (ParameterHandler &prm)
+    {
+      prm.enter_subsection("Material model");
+      {
+        prm.enter_subsection("PerpleX simpler model");
+        {
+	  perplex_file_name   = prm.get ("PerpleX input file name");
+          eta                 = prm.get_double ("Viscosity");
+          k_value             = prm.get_double ("Thermal conductivity");
+          min_temperature     = prm.get_double ("Minimum material temperature");
+          max_temperature     = prm.get_double ("Maximum material temperature");
+          min_pressure        = prm.get_double ("Minimum material pressure");
+          max_pressure        = prm.get_double ("Maximum material pressure");
+        }
+        prm.leave_subsection();
+      }
+      prm.leave_subsection();
+
+      // Declare dependencies on solution variables
+      this->model_dependence.viscosity = NonlinearDependence::none;
+      this->model_dependence.thermal_conductivity = NonlinearDependence::none;
+      
+      this->model_dependence.density = NonlinearDependence::temperature
+	                               | NonlinearDependence::pressure
+	                               | NonlinearDependence::compositional_fields;
+      this->model_dependence.compressibility = NonlinearDependence::temperature
+                                               | NonlinearDependence::pressure
+                                               | NonlinearDependence::compositional_fields;
+      this->model_dependence.specific_heat = NonlinearDependence::temperature
+                                             | NonlinearDependence::pressure
+                                             | NonlinearDependence::compositional_fields;
+    }
+  }
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    ASPECT_REGISTER_MATERIAL_MODEL(PerpleXSimpler,
```

Comment:
I don't like the name "simpler". Can you think of something better? PerplexLookup?